src/lib/openapi/spec/project-overview-schema.ts

```diff
@@ -17,42 +17,60 @@ export const projectOverviewSchema = {
     properties: {
         stats: {
             $ref: '#/components/schemas/projectStatsSchema',
+            description: 'Project statistics',
         },
         version: {
             type: 'number',
+            example: 1,
+            description: '',
         },
         name: {
             type: 'string',
+            example: 'dx-squad',
+            description: 'Name of this project used to identify it',
```

Comment:
I think we should remove the "used to identify it" part, because it makes it sound like the project ID, which it is not, right?

```suggestion
            description: 'The name of this project',
```